main.py

```diff
@@ -1,7 +1,22 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [("George", "Klooney"), ("Kianu", "Reaves"),
+              ("Scarlett", "Keegan"), ("Will", "Smith"),
+              ("Jaden", "Smith"), ("Scarlett", "Johansson")]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    for first_name, last_name in actors:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(first_name="Kianu", last_name="Reaves")\
+        .update(first_name="Keanu", last_name="Reeves")
```

Comment:
```suggestion
    (
        Actor.objects
        .filter(first_name="Kianu", last_name="Reaves")
        .update(first_name="Keanu", last_name="Reeves")
    )
```

Reply:
![image](https://user-images.githubusercontent.com/110391080/212497170-57d16a0c-d0e4-4e5e-be3b-3624efa1f630.png)
doesnt work

Reply:
Take a look at indentation of Olena’s suggestion and your own, it’s different. Also, you can split lines in this way:
```
    Actor.objects.filter(
         first_name="Kianu", last_name="Reaves"
     ).update(first_name="Keanu", last_name="Reeves")

```